x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/single_metric_config.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SYNTHETICS_STEP_NAME } from '../constants/field_names/synthetics';
+import { ConfigProps, SeriesConfig } from '../../types';
+import { FieldLabels, FORMULA_COLUMN } from '../constants';
+import { buildExistsFilter } from '../utils';
+
+export function getSyntheticsSingleMetricConfig({ dataView }: ConfigProps): SeriesConfig {
```

Comment:
This seems to be a misnomer at the moment as this only supports availability metric, rather than multiple types of synthetics single metrics. Will this be expanded to include more single metric options? If not, the name should probably be more specific.

Reply:
Yes we will add more configs into it.